src/manyRubrics/Assignment.java

```diff
@@ -3,22 +3,34 @@
 public class Assignment {
 
 	private String name;
-	private double score;
 	
 	public Assignment(String name) {

```

Comment:
Maybe add a max score for an assignment so then you can compare what score a student got to what the max score can be.